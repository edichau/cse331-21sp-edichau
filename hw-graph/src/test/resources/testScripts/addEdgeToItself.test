#test if you can add an edge from one node to itself

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1

# Add an edge
AddEdge graph1 n1 n1 e1
