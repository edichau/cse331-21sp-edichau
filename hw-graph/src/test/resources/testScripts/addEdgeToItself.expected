#test if you can add an edge from one node to itself

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1

# Add an edge
added edge e1 from n1 to n1 in graph1
